src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,148 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.TodoFormatException;
+import duke.exception.DeadlineFormatException;
+import duke.exception.EventFormatException;
+import duke.exception.EmptyTasklistException;
+import duke.exception.DoneFormatException;
+import duke.exception.InvalidTaskIdException;
+import duke.exception.TaskAlreadyDoneException;
+import duke.exception.DeleteFormatException;
+
+import duke.ui.DukeInterface;
+import duke.local.DataManager;
+import duke.task.TaskManager;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private final Scanner in;
+    private final DukeInterface dukeUI;
+    private final DataManager dataMgr;
+
+    private boolean isRunning;
+
+    private final String LIST_CMD = "list";
+    private final String HELP_CMD = "help";
+    private final String ADD_TODO_CMD = "todo";
+    private final String ADD_DEADLINE_CMD = "deadline";
+    private final String ADD_EVENT_CMD = "event";
+    private final String SET_TASK_DONE_CMD = "done";
+    private final String DELETE_TASK_CMD = "delete";
+    private final String TERMINATE_CMD = "bye";
+
+    private final String FILE_PATH = "data/duke.txt";
+    //private final String FILE_PATH = "test/test.txt";
```

Comment:
you should get rid of dead code/commented code (they dont add any value to your code)